lib/discover/cache.go

```diff
@@ -124,7 +124,7 @@ func (m *cachingMux) Lookup(deviceID protocol.DeviceID) (addresses []string, err
 	l.Debugln("lookup results for", deviceID)
 	l.Debugln("  addresses: ", addresses)
 
-	addresses = util.UniqueStrings(addresses)
+	addresses = util.UniqueTrimmedStrings(addresses)
```

Comment:
We used to sort these now we don't. I'm not sure that matters other than aesthetically (they are shown in that order in the UI) but I see no reason to change it in this PR